sound/soc/sof/intel/intel.c

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Authors: Guennadi Liakhovetski <guennadi.liakhovetski@linux.intel.com>
+
+/*
+ * Intel-specific SOF code
+ */
+
+#include <linux/device.h>
```

Comment:
@lyakh intel.c  seems very generic. Can this name be a bit more specific? intel-ipc.c or intel-mailbox.c?

Reply:
for now this file only contains IPC / mailbox code, but in principle it's thought as a place for any Intel SOF code. But if we want separate files, we can name this one more specifically.